src/common/util.c

```diff
@@ -268,15 +268,21 @@ char *oscap_basename(char *path)
 char *oscap_dirname(char *path)
 {
 #ifdef _WIN32
```

Comment:
Similarly, I would move `ifdef` blocks out of the function, so there would be two definitions, one for win32 and one for others.

Reply:
why? personal preference?

Reply:
@mpreisler I have already changed that. It looks more beautiful now.

Reply:
*rolls eyes*